static/js/theme-map/ThemeMap.js

```diff
@@ -153,6 +153,9 @@ class ThemeMap extends ANSWERS.Component {
         this.config.zoomEndListener(this.map.getZoom(), zoomTrigger);
       });
       map.setCanvasClickHandler(() => this.config.canvasClickListener());
+      map.setLoadHandler(() => {
+        this.core.storage.set(StorageKeys.MAP_LOADED, true)
```

Comment:
I guess there's no use to track loading event and set this back to false?

Reply:
MAP_LOADED is only meant to trigger once if that's what you're asking

Reply:
ah I see, I was thinking "load" event more of it loading tiles when the map is moved. Okay!